### PR TITLE
Make test project target .net core and framework

### PR DESCRIPTION
**Pull request by frederik-h**
In PR #74, @TChatzigiannakis mentioned that his tests don't work because they are no longer compatible with dotnet core. I have replaced the test project with a dotnet core style project
which targets both dotnet core and dotnet framework.

You can run the dotnet core tests with `dotnet test Tests.csproj /p:TargetFramework=netcoreapp2.0`.
For the dotnet framework tests, you should replace `netcoreapp2.0` by `net461`.

### CODE REVIEW

File: src/LLVMSharp.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.1</TargetFramework>
+    <TargetFrameworks>netstandard1.6</TargetFrameworks>
```
Comment (frederik-h) [inline src/LLVMSharp.csproj]: I don't know if the dependency on the higher version is ok? Probably ...

Comment (mjsabby) [reply src/LLVMSharp.csproj]: Was there a need to do this?

Comment (frederik-h) [reply src/LLVMSharp.csproj]: :thinking:  I think it should not be strictly necessary to use a higher version. I have initially created and tested the project on Linux with dotnet core. In this environment, everything worked with netstandard1.1. When I later tested it on Windows with the full framework, the project did not build due to missing references. I must admit that using netstandard1.6 was kind of a quick fix. I suppose that I could also add explicit package references to fix the problem. I will check it next week.

Comment (frederik-h) [reply src/LLVMSharp.csproj]: I have now verified that it is not necessary to increase the version. I had some problems when I switched the target framework without cleaning everything (including the nuget package cache). Perhaps that caused the problems ...

**Comment by frederik-h** [comment]
@mjsabby Is this change still relevant now that the OOAPI has been merged (for the master branch) or should I close the PR?

**Comment by TChatzigiannakis** [comment]
> Is this change still relevant now that the OOAPI has been merged (for the master branch) or should I close the PR?

Yes, I think this PR isn't relevant anymore. However, if you could find a way to move files around so that we can run the tests with just `dotnet test` instead of specifying the project file, that would be very welcome I think.

**Pull request closed without merging**